torch/testing/_asserts.py

```diff
@@ -208,7 +194,7 @@ def _trace_mismatches(actual: Tensor, expected: Tensor, mismatches: Tensor) -> _
     rel_diff = abs_diff / torch.abs(b_flat)
     max_rel_diff, max_rel_diff_flat_idx = torch.max(rel_diff, 0)
 
-    return _Trace(
+    return SimpleNamespace(
```

Comment:
I've opted for a namespace over a named tuple in order to prevent access by index. This should make us a little more future proof. Starting with Python 3.7 we should use a dataclass here with a custom `__str__` method to get a good error message as base.